accountsplus/urls.py

```diff
@@ -1,30 +1,29 @@
-from __future__ import unicode_literals
-
 import django.conf.urls
-from django.contrib.auth.urls import url
+import django.contrib.auth.urls
 import django.contrib.auth.views
 
-import views
-import forms
+
+from accountsplus import views
+from accountsplus import forms
 
 urlpatterns = [
-    url(r'^logout/$', views.logout_then_login, name='logout'),
-    url(r'^password_change/$', views.password_change, name='password_change'),
-    url(r'^password_reset/$', views.password_reset, name='password_reset'),
-    url(r'^reset/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
-        django.contrib.auth.views.password_reset_confirm, name='password_reset_confirm'),
+    django.conf.urls.url(r'^logout/$', views.logout_then_login, name='logout'),
+    django.conf.urls.url(r'^password_change/$', views.password_change, name='password_change'),
+    django.conf.urls.url(r'^password_reset/$', views.password_reset, name='password_reset'),
+    django.conf.urls.url(r'^reset/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
+        django.contrib.auth.views.PasswordResetConfirmView, name='password_reset_confirm'),
```

Comment:
@waseem-omar Please add `.as_view()` to `PasswordResetConfirmView`

Reply:
Nice catch!